build.gradle.kts

```diff
@@ -454,7 +454,6 @@ tasks.register("python37PostCommit") {
   dependsOn(":sdks:python:test-suites:direct:py37:directRunnerIT")
   dependsOn(":sdks:python:test-suites:direct:py37:hdfsIntegrationTest")
   dependsOn(":sdks:python:test-suites:direct:py37:azureIntegrationTest")
-  dependsOn(":sdks:python:test-suites:direct:py37:mongodbioIT")
```

Comment:
Do we need to remove it from postcommits if manually setting the container works?

Reply:
Just because we already run same test ":sdks:python:test-suites:direct:mongodbioIT" in beam_PostCommit_Python_MongoDBIO_IT and I consider no need duplicate run here.